ods_ci/tests/Tests/200__monitor_and_manage/200__metrics/205_bias_metrics.robot

```diff
@@ -0,0 +1,199 @@
+    Create Metrics request via CLI     metrics_type=dir   modelId=${modelId}    protectedAttribute="customer_data_input-3"
+    ...       favorableOutcome=0   outcomeName="predict"    privilegedAttribute=1.0    unprivilegedAttribute=0.0
+    Verify TrustyAI Metrics Exists In Observe Metrics    trustyai_dir    retry_attempts=2
+
+*** Keywords ***
+Bias Metrics Suite Setup
+    [Documentation]    Setup to configure TrustyAI metrics
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+    IF    ${SETUP_MONITORING} == ${TRUE}
+        Log    Enabling and Configuring User Workload Monitoring
+        Enable User Workload Monitoring
+        Configure User Workload Monitoring
+        Verify User Workload Monitoring Configuration
+    END
+    Launch Data Science Project Main Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    Create S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=model-serving-connection
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=${aws_bucket}
+    Create Model Server    token=${FALSE}    server_name=${RUNTIME_NAME}
+
+
+Enable User Workload Monitoring
+    [Documentation]    Enable User Workload Monitoring for the cluster for user-defined-projects
+    RHOSi Setup
+    ${return_code}    ${output}    Run And Return Rc And Output   oc apply -f ${MONITORING_CONFIG_FILEPATH}
+    Log To Console    ${output}
+    Should Be Equal As Integers    ${return_code}     0   msg=Error while applying the provided file
+
+Configure User Workload Monitoring
+    [Documentation]    Configure the retention period in User Workload Monitoring for the cluster.
+    ...                This period can be configured for the component as and when needed.
+    ${return_code}    ${output}    Run And Return Rc And Output   oc apply -f ${UWM_CONFIG_FILEPATH}
+    Log To Console    ${output}
+    Should Be Equal As Integers    ${return_code}     0   msg=Error while applying the provided file
+
+Verify User Workload Monitoring Configuration
+    [Documentation]    Verifies that the ALL monitoring components for user-defined-projects
+    ...    are READY in the ${DEFAULT_MONITORING_NS} namespace
+    [Tags]    Smoke
+    ...       Tier1
+    ...       OpenDataHub
+    Wait For Pods Status  namespace=${DEFAULT_MONITORING_NS}  timeout=60
+    Log  Verified Applications NS: ${DEFAULT_MONITORING_NS}  console=yes
+
+Verify trustyai-service-operator-controller-manager Deployment
+    [Documentation]    Verifies the correct deployment of the model controller in the namespace
+    [Arguments]    ${num_replicas}=1
+    ${all_ready} =    Run    oc get deployment -n ${APPLICATIONS_NAMESPACE} trustyai-service-operator-controller-manager | grep ${num_replicas}/${num_replicas} -o  # robocop:disable
+    Should Be Equal As Strings    ${all_ready}    ${num_replicas}/${num_replicas}
```

Comment:
you could try using `Wait For Pods To Be Ready`, it can checks the number of ready replicas